admin_app/package.json

```diff
@@ -9,24 +9,23 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@heroicons/react": "^2.0.18",
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
+    "@fontsource/roboto": "^5.0.12",
+    "@mui/icons-material": "^5.15.10",
+    "@mui/material": "^5.15.10",
     "jwt-decode": "^4.0.0",
-    "next": "13.5.4",
+    "next": "14.1.3",
     "react": "^18",
-    "react-dom": "^18",
-    "react-markdown": "^9.0.1",
-    "react-textarea-autosize": "^8.5.3",
-    "tailwindcss": "^3"
+    "react-dom": "^18"
   },
   "devDependencies": {
-    "@types/node": "^20.8.10",
+    "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "autoprefixer": "^10",
     "eslint": "^8",
-    "eslint-config-next": "13.5.4",
-    "postcss": "^8",
-    "prettier": "^3.0.3",
```

Comment:
Where did prettier go?
We probably need it right?

Reply:
Not sure, maybe something went wrong with my rebase? Please add it back if you can